### PR TITLE
Registration Page: Admin Registration Page

### LINKED ISSUE #81: Frontend: Registration Page
When I am registering and login in for the first time, I want to choose the name and the class or company that I am part of, depending if I am student, a mentor or admin, or skills if I am a mentor, so I can use the web app accordingly to my role.

Create a page below that considers the data retrieved with the endpoint developed on the issue to show the list #58 and the page component created on this issue #61 - RegistrationPage - src/client/containers/RegistrationPage/RegistrationPage.component.js

If student the app should show this:
image

If mentor the app should show this:
image

If admin the app should show this:
image

IMPORTANT: make sure you use existing components, if needed change the existing components or work together with the developer that created those same components.

### PR DESCRIPTION
**Pull request by Deepti-Dk**
# Description

Admin registration page created using existing components.

Fixes #81 (part 2 of 3)

# How to test?

Tested on storybook - working and responsive

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by bytewiz — DISMISSED** [review]
Is it supposed to delete the `Registration page`? 
Also there is merge conflict to be solved before I can approve it 👍

**Comment by Deepti-Dk** [comment]
> Is it supposed to delete the `Registration page`?
> Also there is merge conflict to be solved before I can approve it 👍

3 separate pages created instead of 1 registration page. updated the route name to be consistent with other 2 pages, merge conflict solved.

File: src/server/api/routes/api-router.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ const TopicsRouter = require('./topics.router');
 
 const skillsRouter = require('./skills.router');
 
-
```
Comment (tpetrina) [inline src/server/api/routes/api-router.js:12]: Undo this file to prevent another merge conflict.

**Review by tpetrina — COMMENTED** [review]
Fix merge conflict - You have a potential for merge conflicts since this PR touches on another files.

**Comment by Deepti-Dk** [comment]
> Fix merge conflict - You have a potential for merge conflicts since this PR touches on another files.

Done

**Comment by Deepti-Dk** [comment]
> Fix merge conflict - You have a potential for merge conflicts since this PR touches on another files.

Done

**Review by tpetrina — APPROVED** [review]

**Review by nuha90 — DISMISSED** [review]

**Review by nuha90 — APPROVED** [review]

File: src/client/App.js [hunk 2]
```diff
@@ -43,8 +43,8 @@ function App() {
             <Route exact path="/selection">
               <SelectionPage />
             </Route>
-            <Route exact path="/registration/student">
-              <StudentRegistrationPage />
+            <Route exact path="/registration/admin">
+              <AdminRegistrationPage />
```
Comment (tpetrina) [inline src/client/App.js:47]: @Deepti-Dk You removed the previous student registration. You should have both, right?

**Review by tpetrina — CHANGES_REQUESTED** [review]

File: src/client/App.js (hunk 2 above)
Comment (Deepti-Dk) [reply src/client/App.js:51]: yes, added again

**Review by tpetrina — APPROVED** [review]

**Review by MaherHussain — APPROVED** [review]

**Pull request merged**